account_asset_management/README.rst

```diff
@@ -0,0 +1,46 @@
+Financial asset management.
+===========================
+
+This Module manages the assets owned by a company. It will keep
+track of depreciation's occurred on those assets. And it allows to create
+accounting entries from the depreciation lines.
+
+The full asset life-cycle is managed (from asset creation to asset removal).
+
+Assets can be created manually as well as automatically
+(via the creation of an accounting entry on the asset account).
+
+Excel based reporting is available via the 'account_asset_management_xls' module.
+
+The module contains a large number of functional enhancements compared to
+the standard account_asset module from Odoo.
+
+Known issues
+------------
+
+The module in NOT compatible with the standard account_asset module.
+
```

Comment:
Missing bug tracker link (this is brand new)
see template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst